src/network/clientpackethandler.cpp

```diff
@@ -1161,6 +1161,23 @@ void Client::handleCommand_HudSetSky(NetworkPacket* pkt)
 	m_client_event_queue.push(event);
 }
 
+void Client::handleCommand_HudSetClouds(NetworkPacket* pkt)
+{
+	std::string datastring(pkt->getString(0), pkt->getSize());
+	std::istringstream is(datastring, std::ios_base::binary);
+
+	u16 density                      = readU16(is);
+	video::SColor *color             = new video::SColor(readARGB8(is));
```

Comment:
…and unions can't have members with constructors :stuck_out_tongue: I added the `delete`.